ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -142,7 +158,10 @@ final class BalanceViewController: BaseViewController {
     }
     
     @objc func accountBtnHandle() {
-        let accountViewController = AccountViewController()
-        navigationController?.pushViewController(accountViewController, animated: true)
+        presentAccountVC()
+    }
+
+    func presentAccountVC() {
+        flowDelegate?.presentAccountController(in: self)
```

Comment:
tady bych změnil naming té metody. Viděl bych radši pojménování "co se stalo" a ne "co se má stát". Kdybys to chtěl použít na více místech, tak nemusíš výdycky chtít presentovat AccountVC

Reply:
Jo o tomhle vím. Já to prezentuji ze dvou míst s nechtěl jsem vytvářet dvě totožné metody. Jedno bylo předtím requestError a druhé bylo buttonTapped.

Reply:
Jak mám tedy tohle vyřešit, když ze dvou míst pushnuji nový stejný VC?

Reply:
@LukasHromadnik rf

Reply:
za mě nejlepší každou z těch akcí udělat zvlášť na flow delegáta a až v něm zavolat jednu metodu, která to neprezentuje